profile_xf05id1/startup/15-optics.py

```diff
@@ -2,9 +2,24 @@
 from ophyd import Device
 from ophyd import Component as Cpt
 
+class SRXSlitsWB(Device):
+    #THIS IS BROKEN AT EPICS :LEVEL
```

Comment:
Perhaps this is not the example @klauer had in mind, but I suspect there are others. Maybe he can chime in to remind you of whatever issue is seems you two discussed.